chromadb/test/db/test_system.py

```diff
@@ -727,53 +729,54 @@ def test_update_segment(sysdb: SysDB) -> None:
 
     # TODO: revisit update segment - push collection id
 
-    result = sysdb.get_segments(id=segment["id"])
+    result = sysdb.get_segments(id=segment["id"], collection=segment["collection"])
     result[0]["collection"] = segment["collection"]
     assert result == [segment]
 
-    result = sysdb.get_segments(id=segment["id"])
+    result = sysdb.get_segments(id=segment["id"], collection=segment["collection"])
     result[0]["collection"] = segment["collection"]
     assert result == [segment]
 
     # Update collection to new value
-    segment["collection"] = sample_collections[1]["id"]
-    sysdb.update_segment(segment["id"], collection=segment["collection"])
-    result = sysdb.get_segments(id=segment["id"])
-    result[0]["collection"] = segment["collection"]
-    assert result == [segment]
+    # TODO: not currently supported
+    # segment["collection"] = sample_collections[1]["id"]
+    # sysdb.update_segment(segment["id"], collection=segment["collection"])
+    # result = sysdb.get_segments(id=segment["id"], collection=segment["collection"])
+    # result[0]["collection"] = segment["collection"]
+    # assert result == [segment]
```

Comment:
this section of the test was never asserting what it claims
although it looks correct on first glance, updating the `collection_id` on a segment is actually not supported on the backend (there's a todo)
the reason this test was passing was because of this line:

```python
result[0]["collection"] = segment["collection"]
```

this stopped working when I began filtering returned segments by the updated collection ID, so decided to just comment this out for now

Reply:
We likely never need to support changing the collection of the segment? Should we either

- Explicitly not support this and throw an error
OR
- leave it in and assert that it does nothing / has some defined behavior

Removing the test feels like a way to underspecify the system for future

Reply:
sure, I can update the proto to make it clear this isn't supported